File: 1.FDP/3° Competencia/Java/Seleccion/Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular la convercion de pesos a dolares.\n");
		double Pesos, Dolares, Convercion;
		System.out.println("Pesos: ");
		Pesos= scan.nextDouble();
		if (Pesos > 0) {
			System.out.println("Valor del dolar: ");
			Dolares= scan.nextDouble();
			if (Dolares > 0) {
				Convercion= Pesos/Dolares;
				System.out.println("Cantidad de dolares= " + Convercion);
			} else {
				System.out.println("Error en el valor del dolar");
			}	
		} else {
			System.out.println("Error de pesos");
		}
		//
	}
}

File: 1.FDP/3° Competencia/C++/Seleccion/Ejercicio5.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Convertir pesos a dolares.\n";
	double Pesos, ValorDolar, Dolares;
	cout << "Ingresar la cantidad de pesos:";
	cin >> Pesos;
	if (Pesos > 0) {
		cout << "Ingresar el valor del dolar:";
		cin >> ValorDolar;
		if (ValorDolar > 0) {
			Dolares= Pesos/ValorDolar;
			cout << "Cantidad de dolares = " << Dolares << "\n";
		} else {
			cout << "Error del valor del dolar.\n";
		}
	} else {
		cout << "Error de los pesos.\n";
		
	}
	return 0;
}